pkg/compactor/blocks_cleaner.go

```diff
@@ -402,17 +411,28 @@ func (c *BlocksCleaner) deleteBlocksMarkedForDeletion(ctx context.Context, idx *
 
 // cleanUserPartialBlocks delete partial blocks which are safe to be deleted. The provided partials map
 // and index are updated accordingly.
```

Comment:
`partials` is no longer updated:
```suggestion
// cleanUserPartialBlocks deletes partial blocks which are safe to be deleted. The provided index is updated accordingly.
```